sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePlugin.scala

```diff
@@ -18,8 +18,14 @@ object ScalaNativePlugin extends AutoPlugin {
 
     val nativeClang = settingKey[File]("Location of the clang compiler.")
 
+    val nativeClangOptions = settingKey[Seq[String]](
+      "Additional options are passed to clang during compilation.")
+
     val nativeClangPP = settingKey[File]("Location of the clang++ compiler.")
 
+    val nativeClangPPOptions = settingKey[Seq[String]](
+      "Additional options are passed to clang++ during compilation.")
+
```

Comment:
Those are literally duplicated with `nativeCompileOptions`, they should not be there.